News post for playtest-20201213. (#500)
OpenRA/OpenRAWeb: content/news/playtest-20201213.md

Just in time for the holiday period, we are kicking off a new playtest series for the next OpenRA release.

<img src="/images/news/20201213-christmas.gif" alt="Happy Holidays from the OpenRA team!" width="700px">

Recent development has focused on building the foundation for future features, including a lot of work towards the Tiberian Sun mod which we described in our [last news post](/news/devblog-20200629/). But this does not mean that this release is short on new features!

Some of the new changes to look out for in [playtest-20201213](/download/) include:

* New lobby options for Stealth Deliveries in Tiberian Dawn and Automatic Concrete in Dune 2000

> * Disable spawn points in a game lobby by right clicking on the map preview

Reviewer: ```suggestion
* Disable spawn points in a game lobby by right-clicking on the map preview
```

https://www.merriam-webster.com/dictionary/right-click

* Share OpenRA status and game invites on Discord
* More fluid infantry combat after fixing several long-standing combat bugs
* Battlefield zooming is now centered on the mouse cursor
* Red Alert Spies can now reset support powers by infiltrating the structure that grants them
* Servers can be configured to save replays, enabling the creation of community ladders

> Red Alert and Tiberian Dawn include an updated set of skirmish/multiplayer maps, and community balance changes refresh their multiplayer meta. Dune 2000 features new weapon damage calculations that are more faithful to the original game, plus a number of other small consistency fixes. For single-player fans, Red Alert includes 8 more missions from the original game, and the AI behaviour and performance has been improved across the board.

Reviewer: ```suggestion
Red Alert and Tiberian Dawn include an updated set of skirmish/multiplayer maps, and community balance changes refresh their multiplayer meta. Dune 2000 features new weapon damage calculations that are more faithful to the original game, plus a number of other small consistency fixes. For single-player fans, Red Alert includes 8 more missions from the original game, and the AI behavior and performance has been improved across the board.
```

as you wrote `centered` in American English above.

Our main macOS release now requires 10.13 or newer, but we plan to keep supporting players on 10.9 &ndash; 10.12 using a new "compat" release that can be downloaded from our [GitHub Release Milestones](https://github.com/OpenRA/OpenRA/releases).

This post has covered just a small subset of the [full changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog/d207b07fe1713e2dedf547dbb09e03d120cffe9b), and we encourage anybody interested in finding out more to read the full list. Head on over to the [download page](/download/) to install the new playtest, and let us know what you think in the comments below, on [our forum](https://forum.openra.net/), [community Discord server](https://discord.gg/UcUztfr), or [GitHub](https://github.com/OpenRA/OpenRA/issues).

> Remember that playtests install along side the OpenRA release version, and you can automatically switch between the two versions when joining a multiplayer server, so there is no reason not to give it a try!

Reviewer: ```suggestion
Remember that playtests install alongside the OpenRA release version, and you can automatically switch between the two versions when joining a multiplayer server, so there is no reason not to give it a try!
```
